extensions/GitExtension/FileExplorerGitIntegration/Models/GitLocalRepository.cs

```diff
@@ -72,89 +72,54 @@ IPropertySet ILocalRepository.GetProperties(string[] properties, string relative
             switch (propName)
             {
                 case "System.VersionControl.LastChangeMessage":
-                    if (latestCommit == null)
+                    latestCommit ??= FindLatestCommit(relativePath, repository);
+                    if (latestCommit != null)

```

Comment:
Nit: use "is not null" which shortcuts overloaded operators for the null case. Same comment throughout for other null comparisons.

Reply:
> Nit: use "is not null" which shortcuts overloaded operators for the null case. Same comment throughout for other null comparisons.

Wow! Today I Learned...